README.md

```diff
@@ -2,38 +2,33 @@
 
 [![GitHub Build Status](https://github.com/cisagov/cool-domain-manager-iam/workflows/build/badge.svg)](https://github.com/cisagov/cool-domain-manager-iam/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) [Terraform
-module](https://www.terraform.io/docs/modules/index.html) GitHub
-repository started.  This skeleton project contains [licensing
-information](LICENSE), as well as [pre-commit
-hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
-
-See [here](https://www.terraform.io/docs/modules/index.html) for more
-details on Terraform modules and the standard module structure.
+This project is used to manage IAM permissions for Domain Manager users.
```

Comment:
Do we have a project to link to for "Domain Manager"?

Reply:
[`cisagov/domain-manager`](https://github.com/cisagov/domain-manager) looks like it was intended to be a central repo, but there's nothing really there.  Most of what we care about is in [`cisagov/domain-manager-cicd`](https://github.com/cisagov/domain-manager-cicd)- if I had to pick something to link to, it would probably be that.  Thoughts?

Reply:
I'm reluctantly ok with [cisagov/domain-manager-cicd](https://github.com/cisagov/domain-manager-cicd). Ideally [cisagov/domain-manager](https://github.com/cisagov/domain-manager) would tie the pieces together with [cisagov/domain-manager-cicd](https://github.com/cisagov/domain-manager-cicd), [cisagov/domain-manager-api](https://github.com/cisagov/domain-manager-api), and [cisagov/domain-manager-ui](https://github.com/cisagov/domain-manager-ui) much like [cisagov/cool-system](https://github.com/cisagov/cool-system) does for COOL pieces.

Reply:
Works for me- we can change it later when all of the Domain Manager repos have achieved our lofty standards.  

Link added in https://github.com/cisagov/cool-domain-manager-iam/pull/2/commits/d0e8fafa2ffaee01a9c4c525bc5e1d160e84d882.